javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,33 @@
+console.log("Housey-pricey exercise");
+//peter's house//
+const houseWidth = 8;
+const houseDepth = 10;
+const houseHeight = 10;
+const Gardensize = 100;
+const ActualPrice = 2500000;
+
+let volume = houseWidth * houseDepth * houseHeight;
+const peterprice = volume * 2.5 * 1000 + Gardensize * 300;
```

Comment:
`camelCase` would be better
```suggestion
const peterPrice = volume * 2.5 * 1000 + Gardensize * 300;
```
But even better some more descriptive name as:
```suggestion
const estimatedHousePrice = volume * 2.5 * 1000 + Gardensize * 300;
```